test/acceptance/features/steps/service_binding.py

```diff
@@ -10,3 +10,6 @@ class ServiceBinding(object):
     def create(self, yaml):
         sbr_create_output = self.openshift.oc_apply(yaml)
```

Comment:
Please, replace this line with
```go
sbr_create_output = self.attempt_to_create(yaml)
```
to avoid duplicating pieces of code.

Reply:
Done.